ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/CreateTableCommand.java

```diff
@@ -36,6 +37,11 @@ public CreateTableCommand(CreateTable createTable, Map<String, Object> overridde
 
     Map<String, Expression> properties = createTable.getProperties();
 
+    if (!properties.containsKey(DdlConfig.KEY_NAME_PROPERTY)) {
```

Comment:
Do we need to do additional validation? Like what if the key of the message has a different field name? What if  this key refers to a field in the value? Will that work?

Reply:
I added additional check to make sure the specified key column name exists in the schema.